.github/workflows/rust.yml

```diff
@@ -50,6 +50,9 @@ jobs:
     - name: Install verilator, jq, and SBT
       run: sudo apt-get install verilator jq sbt
 
+    - name: Install numpy
+      run: pip3 install numpy
```

Comment:
But why?

Reply:
because it makes everything so much easier is why. feel free to remove it if you want